src/reducers/__tests__/existingThoughtMove.ts

```diff
@@ -458,23 +457,13 @@ it('move with duplicate descendant', () => {
     }),
   ]
 
-  // run steps through reducer flow and export as plaintext for readable test
   const stateNew = reducerFlow(steps)(initialState())
   const exported = exportContext(stateNew, [ROOT_TOKEN], 'text/plain')
 
-  // contextIndex
   expect(exported).toBe(`- ${ROOT_TOKEN}
   - a
     - b
-      - x
-      - y
-        - x`)
-
-  // x should have contexts a/b and a/b/y
-  expect(getContexts(stateNew, 'x'))
-    .toMatchObject([
-      { context: ['a', 'b'], rank: 0 },
-      { context: ['a', 'b', 'y'], rank: 0 }
-    ])
```

Comment:
You'll want to test `getContexts` as well, since `exportContext` is only touching `contextIndex` while `getThought` or `getContexts` will touch `thoughtIndex`.

Reply:
Sure. That makes sense.